lesson-3/lesson-3.js

```diff
@@ -0,0 +1,145 @@
+//Урок 3
+
+//Задание 1
+//С помощью цикла while вывести все простые 
+//числа в промежутке от 0 до 100
+console.log('Задание 1');
+var primeNumbers = new Array(2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97);
+var quantityNumbers = primeNumbers.length;
+var i = 0;
+
+while(i < quantityNumbers){
+    console.log(primeNumbers[i]);
+    i++
+}
+
+//Задание 2;3.
+//Gодсчета стоимости корзины в зависимости от находящихся в ней товаров
+console.log('Задание 2-3');
+var basket = new Array();
+var products = new Array([1, 'Телевизор', 63990], [2, 'Холодильник', 43890], [3, 'Утюг', 4590], [4, 'Пылесос', 7350]);
+
+function catalogFormation(products){
+    var catalog = 'Каталог \r\n';
+    for(var i=0; i<products.length; i++){
```

Comment:
не забывайте о читаемости кода
```suggestion
    for(var i = 0; i < products.length; i++){
```